app/assets/javascripts/components/image-cropper.js

```diff
@@ -0,0 +1,78 @@
+//= require cropperjs/dist/cropper.js
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function ImageCropper ($imageCropper) {
+    this.$imageCropper = $imageCropper
+    this.$image = this.$imageCropper.querySelector('.app-c-image-cropper__image')
+  }
+
+  ImageCropper.prototype.init = function () {
+    // This only runs if the image isn't cached
+    this.$image.addEventListener('load', function () {
+      this.initCropper()
+    }.bind(this))
+
+    // This should only run if the image is cached
+    if (this.$image.complete) {
+      this.initCropper()
+    }
+
+    this.setupFormListener()
+  }
+
+  ImageCropper.prototype.initCropper = function () {
+    if (!this.$image.complete) {
+      return
+    }
+
+    var width = this.$image.clientWidth
+    var naturalWidth = this.$image.naturalWidth
+    var scaledRatio = width / naturalWidth
+
+    // Adjust the crop box limits to the scaled image
+    var minCropBoxWidth = Math.round(960 * scaledRatio)
+    var minCropBoxHeight = Math.round(640 * scaledRatio)
```

Comment:
Is there a reason you used Math.round() for these values? I think this will cause problems with rounding down, because the minimum crop size will be smaller than 960x640 pixels meaning we couldn't support the image.

Reply:
I copied these over from content publisher, but you're right, have changed to `ceil` and specified the width and height for the canvas